kernel/src/debug.rs

```diff
@@ -104,12 +104,14 @@ pub trait IoWrite {
 pub unsafe fn panic_print<W: Write + IoWrite, C: Chip, PP: ProcessPrinter>(
     writer: &mut W,
     panic_info: &PanicInfo,
-    nop: &dyn Fn(),
+    nop: Option<&dyn Fn()>,
```

Comment:
This seems like a significant API change -- should we discuss?

Reply:
Yeah, I was planning on splitting it out